docs/concepts.mdx

```diff
@@ -7,8 +7,6 @@ description: Core Concepts of Kwil
 slug: /concepts
 ---
 
-# Kwil Concepts
```

Comment:
This is technically a second/duplicate top-level header definition.  The first coming from the `title:` it would seem.  My linter complains, and "Kwil Concepts" still ends up at the top of the page when this is removed.